bundles/org.openhab.binding.paradoxalarm/src/main/java/org/openhab/binding/paradoxalarm/internal/model/ParadoxInformationConstants.java

```diff
@@ -0,0 +1,46 @@
+/**
+ * Copyright (c) 2010-2019 Contributors to the openHAB project
+ *
+ * See the NOTICE file(s) distributed with this work for additional
+ * information.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ */
+package org.openhab.binding.paradoxalarm.internal.model;
+
+import java.util.HashMap;
+import java.util.Map;
+
+/**
+ * The {@link ParadoxInformation} Class that provides the basic panel
+ * information (serial number, panel type, application, hardware and bootloader
+ * versions. It's the object representation of 37 bytes 0x72 serial response.
+ *
+ * @author Konstantin_Polihronov - Initial contribution
+ */
+public class ParadoxInformationConstants {
+    public static Map<String, PanelType> panelTypes = new HashMap<String, PanelType>();

```

Comment:
Please use the diamond operator, ie replace the second occurrence of the same generic by `<>`